<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>370</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="qSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="singleStep">
    <number>25</number>
   </property>
   <property name="pageStep">
    <number>25</number>
   </property>
   <property name="sliderPosition">
    <number>99</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Quality</string>
   </property>
  </widget>
  <widget class="QLabel" name="qSliderLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Best</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="extensionBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>mp4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mkv</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>webm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mov</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wmv</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>avi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mp3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>m4a</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>aac</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wav</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ogg</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushConfirm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushReset">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Extension</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkVerbose">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>178</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Verbose</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSubtitles">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>197</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Download Subtitles</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkMetadata">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>218</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Metadata to JSON</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="retryBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Retries</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rateLimiterBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="suffix">
    <string>KB/s</string>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>50000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>100.000000000000000</double>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
   </property>
   <property name="value">
    <double>500.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rate Limiter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkRate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QLabel" name="headerLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
     <hintingpreference>PreferFullHinting</hintingpreference>
     <fontweight>ExtraBold</fontweight>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Advanced Options</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
